controllers/clusterversion_controller.go

```diff
@@ -106,10 +110,29 @@ func (c *ClusterVersionReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
+	subscriptionPredicates := builder.WithPredicates(
+		predicate.NewPredicateFuncs(
+			func(client client.Object) bool {
+				return client.GetNamespace() == c.OperatorNamespace
+			},
+		),
+		predicate.Or(predicate.LabelChangedPredicate{}, predicate.GenerationChangedPredicate{}),
```

Comment:
Why do we need to queue an event if the subscription spec changes?

Reply:
no, we aren't doing it, above only requeue on label changes.